sqllineage/core/handlers/source.py

```diff
@@ -74,6 +74,12 @@ def _handle_table(self, token: Token, holder: SubQueryLineageHolder) -> None:
                 self._handle(token.tokens[1], holder)
         elif token.ttype == Literal.String.Single:
             self.tables.append(Path(token.value))
+        elif (
+            type(token).__name__ == "Function"
+            and str(token)[0:10].lower() == "openrowset"
+        ):
+            # Handle OPENROWSET in T-SQL (Issue #290) - allows views to SELECT from filesystems on T-SQL
+            pass
```

Comment:
Please rewrite it in a pythonic way , `isinstance(token, Function) and token.normalized.startswith("OPENROWSET")` makes sense to me. 

Also as mentioned in the other comment. Please get the path and make it as source. You can leverage get_parameters method in Function class.